Jenkinsfile

```
@@ -24,52 +24,68 @@ pipeline {
                 sh 'python3 tools/check_format.py --verbose --compare-to origin/main'
             }
         }
-        stage('Setup gc-eu-mq-dbg') {
+        stage('Build gc-eu-mq-dbg') {
             steps {
                 sh 'cp /usr/local/etc/roms/oot-gc-eu-mq-dbg.z64 baseroms/gc-eu-mq-dbg/baserom.z64'
                 sh 'make -j setup'
+                sh 'make -j RUN_CC_CHECK=0'
+                sh 'make clean assetclean'
```

```suggestion
                sh 'make -j setup VERSION=gc-eu-mq-dbg'
                sh 'make -j RUN_CC_CHECK=0 VERSION=gc-eu-mq-dbg'
                sh 'make clean assetclean VERSION=gc-eu-mq-dbg'
```
reasoning:
- consistency with the rest
- we may remove having a default version eventually?